buildsystem/codecompliance/__main__.py

```diff
@@ -140,7 +140,7 @@ def get_changed_files(gitref):
         file_list = subprocess.check_output(invocation)
 
     except subprocess.CalledProcessError as exc:
-        raise Exception(
+        raise SyntaxError(
```

Comment:
raising the general is exception is ok to crash the program imo.

Reply:
I'll change it to `RuntimeError`, see below why.